components/brave_welcome_ui/components/svg/avatar-icon.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+const backgroundFills = ['#DFE1E5',
+'#B7DEFF', '#F5C5E2', '#CCC5F5', '#C5F5DB', '#E9F5C5']
+
+const AvatarIcon = (props: any) => {
+  const bgFill = backgroundFills[Math.round(Math.random() * (backgroundFills.length - 1))]
```

Comment:
Solution is probably to use `React.useMemo` with no dependencies

Reply:
done